.circleci/config/commands/@caches.yml

```
@@ -40,6 +40,8 @@ refresh_go_mod_cache:
     - run:
         name: go mod download
         command: |
+          # set GOPATH explicitly to download to the right cache
+          export GOPATH=/home/circleci/go
```

I wonder if we could rely on `$HOME` rather than hardcoding the path; also ensure the path exists?

```suggestion
          export GOPATH="${HOME}/go"
          mkdir -p "$GOPATH"
```

---

If it doesn't exist, we're going to have a bad time. :) But I'll add the `$HOME` bit.